pxtblocks/blocklyloader.ts

```diff
@@ -362,6 +362,22 @@ namespace pxt.blocks {
                 });
             }
             else {
+                // if requested, wrap block into a "set variable block"
+                if (fn.attributes.blockSetVariable && fn.retType) {
+
+                    const setblock = Blockly.Xml.textToDom(`
+<block type="variables_set" gap="${fn.attributes.blockGap || 8}">
+<field name="VAR" variabletype="">${fn.retType.toLowerCase()}</field>
+</block>`);
+                    {
+                        let value = goog.dom.createDom('value');
```

Comment:
We generally use `document.createElement()`. I'd rather not take a dependency on the Google closure stuff

Reply:
We already use good.dom.createDom in various spots. Cleanup for another PR.